pavelib/tests.py

```diff
@@ -27,9 +27,11 @@
     ("failed", "f", "Run only failed tests"),
     ("fail_fast", "x", "Fail suite on first failed test"),
     ("fasttest", "a", "Run without collectstatic"),
+    ("without_migrations", "m", "Run without migrations"),
```

Comment:
Might want to add "(overrides --settings=)" to be clear about the relationship between them.